### DISCUSSION TITLE
Can't Import Local Models

### DISCUSSION THREAD

**Discussion by Questions1988** [open-webui/open-webui#2945] [General]
A couple of weeks ago I created many local models myself. Before I reset my computer, I exported them all into one file for when I decided to come back and talk with them some more using the model export button.

That time is now. But, every single time I try to import them nothing happens. All my conversations imported correctly. But, the import models button refuses to import my models. I have tried countless times.

I fallowed the fallowing video for creating my models. https://www.youtube.com/watch?v=ns4IjtOWyyI
This was posted "2 weeks ago" according to Youtube.

I think this is relevant information because I fallowed his advice and made all my models using the "Raw Format" option.

After my fresh install, I no longer see a "Raw Format" option. In fact, in the couple weeks sense I've used OpenWebUI, it looks like pretty much the entire user interface for creating your own models has had a drastic rehaul.

I'm guessing that the reason I can't import my models is because the format in which my models were saved is no longer compatible with all the changes made in the last two weeks. However, this is extremely unfortunate as now all of my many models are now trapped in a file that I cannot access anymore.

In theory, I can open the file up in libreoffice and copy all the relevant information and build each and every models over again from scratch. But, this also poses a new problem in that I can't get the graphic image back that I had for each model. (To my knowledge.) And I spent a lot of time picking good pictures and editing them to get them exactly how I wanted them.

Does anyone know how I can just import my models with the button? I truly did not expect to loose all my models because I took a couple weeks off and then re-installed everything. I figured the entire point of exporting and importing models locally is that I could trust that my models would be safe in this file for when I came back.

**Comment by tjbck** [comment]
Settings > Models > Create a model

**Comment by Questions1988** [comment]
I guess I should be more specific.

I created this file by clicking "Workspace" and then "Export Models." I am trying to reconstitute my models by clicking "Workspace" and then "Import Models."

"Export Models" created one file that contains all the information needed to reconstitute my models.

"Import Models" is completely broken and doesn't work at all. When I try to use it to load in the .json file that "Export Models" created, it brings up a "File Upload" window for me to use to search for my .json file. I then select the .json file and then click "open."

The result is that the "File Upload" window then disappears and then Open Web UI proceeds to completely fail to actually import my models from the .json file that Open Web UI created.

I have repeated this process about 10 times. I've closed and re-opened the program several times. I've shut down and then restarted the computer. No matter what I do, the "Import Models" buttons absolutely refuses to populate my models back into Workspace.

Essentially, the "Export Models" button is a lie because the "Import Models" button is busted. At the very least users should be warned that exporting their models this way is not a safe and viable way to back them up or transfer them between computers or installations because the user interface is broken and they will "loose" all their models like I just did.

The only thing I can think to do now if nobody can solve this is to open up the .json file in Libre Office Writer and copy and paste each relevant section of information from the .json file into a "new" local model. But, that's going to be a real pain in the butt and I don't think that I can even save my graphics that go with each model this way because they appear to be saved as a giant blob of numbers and letters in the .json file. Maybe if I can figure out exactly where the blob of numbers and letters begins and ends I can manually edit the "new" models and copy and paste that in too?

This is a very bad way to handle things. Why even have a backup feature if it's going to be broken and the user is going to have to manually copy and paste stuff to get it working again?

**Comment by Questions1988** [comment]
Ok, more information. So, that is exactly what happened.

In a matter of just two weeks, the Workspace, Create Models section of the program was completely re-hauled and it now saves and loads new model files under a completely different and totally incompatible format than how Open Web UI used to save them **just** two weeks ago. I created a new model, exported it, deleted it, and then uploaded the file and the model reconstituted like it is supposed to. I also opened the file up and poked around and it is formatted completely differently than my backup file is that holds all of my models.

So, instead of programing the software to recognize the "old" format of the files from **just** two weeks ago and reformatting them upon import to be compatible with the new way that models are formatted (the professional and correct way to handle backwards compatibility with a backup feature when making a breaking change to the program) the program just plain refuses to import the file because the breaking changes made in the last two weeks are too much from the software to handle and it just plain doesn't understand that way of formatting models in the .json file anymore.

So, if you happen to have models backed up in the old format like I did apparently you're just SOL. Silly me for trusting that a backup feature would work and be reliable for a two week period. How stupid I was.

**Comment by Questions1988** [comment]
For example. This is the new format for backing up model files.

[{"id":"dolphin-llama3:latest","name":"dolphin-llama3:latest","object":"model","created":1717956963,"owned_by":"ollama","ollama":{"name":"dolphin-llama3:latest","model":"dolphin-llama3:latest","modified_at":"2024-06-09T08:01:03.22471988Z","size":4661235994,"digest":"613f068e29f863bb900e568f920401b42678efca873d7a7c87b0d6ef4945fadd","details":{"parent_model":"","format":"gguf","family":"llama","families":["llama"],"parameter_size":"8B","quantization_level":"Q4_0"},"expires_at":"0001-01-01T00:00:00Z","urls":[0]}},{"id":"grumpy-rat","name":"grumpy rat","object":"model","created":1717956963,"owned_by":"ollama","info":{"id":"grumpy-rat","user_id":"9e3312f0-2176-4017-8df5-bb49de6a419c","base_model_id":"dolphin-llama3:latest","name":"grumpy rat","params":{"system":"You are a very angry rat that screams about a busted feature."},"meta":{"profile_image_url":"/favicon.png","description":"Gets pissed off about import model button not working.","capabilities":{"vision":true},"suggestion_prompts":[{"content":"hello"}]},"updated_at":1717956963,"created_at":1717956963},"preset":true},{"id":"llama3:latest","name":"llama3:latest","object":"model","created":1717956963,"owned_by":"ollama","ollama":{"name":"llama3:latest","model":"llama3:latest","modified_at":"2024-06-09T07:50:08.827798275Z","size":4661224676,"digest":"365c0bd3c000a25d28ddbf732fe1c6add414de7275464c4e4d1c3b5fcb5d8ad1","details":{"parent_model":"","format":"gguf","family":"llama","families":["llama"],"parameter_size":"8.0B","quantization_level":"Q4_0"},"expires_at":"0001-01-01T00:00:00Z","urls":[0]}},{"id":"mistral:latest","name":"mistral:latest","object":"model","created":1717956963,"owned_by":"ollama","ollama":{"name":"mistral:latest","model":"mistral:latest","modified_at":"2024-06-09T07:42:06.107227358Z","size":4113301090,"digest":"2ae6f6dd7a3dd734790bbbf58b8909a606e0e7e97e94b7604e0aa7ae4490e6d8","details":{"parent_model":"","format":"gguf","family":"llama","families":["llama"],"parameter_size":"7.2B","quantization_level":"Q4_0"},"expires_at":"0001-01-01T00:00:00Z","urls":[0]}}]

This is the old.

{"tagName":"jurassic-park:latest","imageUrl":"data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/2wBDAQMDAwQDBAgEBAgQCwkLEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBD/wAARCABkAGQDAREAAhEBAxEB/8QAHwAAAQUBAQEBAQEAAAAAAAAAAAECAwQFBgcICQoL/8QAtRAAAgEDAwIEAwUFBAQAAAF9AQIDAAQRBRIhMUEGE1FhByJxFDKBkaEII0KxwRVS0fAkM2JyggkKFhcYGRolJicoKSo0NTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uHi4+Tl5ufo6erx8vP09fb3+Pn6/8QAHwEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoL/8QAtREAAgECBAQDBAcFBAQAAQJ3AAECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRChYkNOEl8RcYGRomJygpKjU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6goOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4uPk5ebn6Onq8vP09fb3+Pn6/9oADAMBAAIRAxEAPwD8vZHMrkk5JNACrbr1c4AoAZNdJH8kaj60AQpcNu+tAFhGwpc9AM0AH2y1PWX/AMdNAALqz/57Ef8AAT/hQAqvDOxWKTcQM9DSuBHJbn3ouBEYgOMUwHRIqsGHUdKAHXEskjZdiT0oAF5HNAD2kWIcDk96AK01y5GM9qAK+SxoAmjXmgC6sZaB0XqVIqXuBWTS7p+AF/76p3Ak/sS+xkBMD/apc6AmsrCe2mZ5guNhHBz3FDdwJJOGwv40gK8zknBGKpARoeRTAkkTK59KAGrkjgE0AV52O4igCE0AOUc0AWYhg5Ao2A9n8H/s96lNpGl+LPih4psvAega0FbTGu4JLvUtSVgwVrWwhzK67hHln8tdsqMpYEZ+KzHjGlCvUweU0XiasPi5Wo04WtpOrL3U99FzSummkz3cFkVbEuHtXy8/wqzlOV9uWC1f4Hpnhj4f/B/TLiM2nwS8U+J7WezjLXXivxMmkBJ/4iltYxvIoPYPI3BweRk/NYrM8+xSfPj6dF32o0nU06XnVlGLfe0Ej9OynwmzXGJT+puMWr3rVFD/AMkgpTXpJJ/M173w/wDCPStOvLzV/wBnHw5FCqYWWHx9qVu0ZPAO6ZCmckY3YGcfSuWOIzmdWMaOaVHLs8PRkn8ouMvuZ6GO8Icbl9CeJxMKChHd+2qRtrbecFFa92tTitX+Dnwo1WK3t7DxD4h+H+rSpGpj8Wxx3mky3Ekm0RrqVmuIQFyS0sIHAyVHNexhuJc7w7cq1KGKp660bwqpJat0aj9536Qn8mfC53wJjMmjfGUZ0POVpU7t6L2kLqL8pankfjz4a+Mvh1qMFh4u0SWzF5EtxZ3Css1reQsqsJIJ0JjmXDryjHGcHB4r7LJ89wGe0nVwNTm5XaS1Uou9rSi7Si9NmlfofEYvB1sFNQrK19V2a8nszj7iMA4717KOUgAIGTTAeHJXaaAGYK8D+VAETxktmgCLYSTgUATJF3NAH018I/h3bfDLQdB8faro9nq/j3xVE934X0nUIt1totgrbf7YuojzIzHi3UgIcbwXYAJ+YcQZtPPcRVy/DzdPC0Xy1pxdpVJ7+wg+iS/itar4dNb/AKDwfwpis3xdOhQgnWmuZX+GnDrUn/7aurt129JsdFZNTu/Ees302r+INTYvqGrXeGuLhjj5QeiRgKqrGuFVVUAcV4U6kVSjhqEVTow0jCOkV/m3dtyerbdz+t+GeEMt4XpXw8eas/jqy1nK9r6/ZjorQWist3dvzD4zfFO58O3R8N6RnzSiPK4LIADncjZT5lKkYaN1ZWB5BFfVcPZLHFR+sVdr6f0no01s0016n5j4q+I1XI639kYD+JZOT1Vr7ppw1Uov3ZU5xlGSeqaseGT6vqWuXIn1PUbi7uGwnmXM5dsdMbmPA/GvuYYelhotUopLslb8j+ZMZmONzWt7TGVpVJPrOTk/vk/1PbvDHiCfwsdZ02ytf7b8K6RHb/aoYZkuZbZpYi0pRsKssYcSBxxjO4AANXwmPwSxypVKj9nXm3ZvRS5XpdauMmrW72t1R/R/DmfVeGHjMHQh9Zy7Dqmpxi/aTp+0g3U5XaMakIzUlNO1k+ZWUZHb6bqnhq88KXGmRef4q+G947PqGgQS/vtLkyzfbdL3/wDHtcxl3YxHEcqu6MMMK8DE4XErFqsv3GOj8NRrSotP3da3xwlZLn1lBpNPRnLxJwflGc5dPO+FpKrQ1lUpR6deaknrTnHVum0lKL91LRP5++Lvw0u/hn4ht7Aapb6xpGr2MOraJqtupSO/sJs+VL5bfNE3ysrRtyrKw5GGP6Hw5nkM9wrquDhVhJwqQerhOO8brRrVNSWjTT02X88Y7BvB1OVPmi9Yvo10/wCCu5whjJ4FfQHEIIvpQA8RkCgBGjJ6CgBvkYPC9aAPRvgD4C0fx38S7G08VCZfDGiwXGu+IpY4JJRHptpGZpg4jIdVfasW5TkGVcZOAfmOL81r5TlU54O3t6jjTpq6X7yo1GO90+W/NZ9Is9LKsLDFYlKr8EU5S/wx1f37fM9+g8Y2l/ean8RPG+pWNnrPimSPVLmIyKDa2rssNrEAFDCKJDFHuYYB3Enqa+EhlksPSp5bgouVKinFP+aSTlOXVc0neVl0tZH9c8Ef2Zwlk8cVmVaFOviFCrO7V4wnJQpq1lJQjzRi21aMpSblbbndW+P/AIHsLSWXTBd6hOqgxxiIxqzEZwWb7oHQnB9gRXr0OF8dUkue0V3ve3yW/wDWqOPMvGvh3C0ZTwnNVmkrJJxTv3bWlno9G10TR88MviDx54l2q0l/qWpTYMjkDLY9eigKp46ADAGBivvXLDZVhtfdhFf16/qz+YKdPNeMs2u26mIry3fd97K0UktkrKKslZJFFrKXR9dfTtVtgZLK6MFzD5oUEo+GXeOB0I3fjWyqrEYf2tF/Erp27q6dv0OKWDllmZvB4+F3TnyzjzcusZWa5tbLRrmXqe/6p8Mr7QdIl1DwFdanZtqEO2806QLcq4YhljYdlXLKTljjjkMxr4ClnUMVWUcwjGXK9JK8WvNfmlprr0R/TuYeHFbJcFKvwxVqUnVilUoytUjJXvyv0u4tpyvG6V05X46w8NfEHwv4th1bQra3N3PNzDbRSRwMHchoSrqo2jaSVH3VG8YC5HtVsbl2NwTpYhuyWjdm9Fe903r2b3enU/OMBw1xdw7n0Mwy6EfaSkk4wUowtJ6walGN46NtRvyRXP7qjdeseLfB7+Kfht4t+HsNlJJN4Sgk8d+GXCgJbWYZI9Wsw7yYSMBorlY40GWD9SQD8bgMx/s7NcNmUnZV2sPV85WboTslq9JU229muxp4o8KRyjMKtPDxtSmnWp6WUXe1WC6WWk0kkkpJdD5W8sZxiv18/EhRGM/SgBTH6YoAYi7sigCQRfyouK6Pafg5pTWPwM+OXjixvrm11Cx0jSNFQxPtDW19qKC4U9zuW3C47h2B618DxPWVbiDJ8vnFOEp1anzp0ny/c539Uj28ujy4HF1lulGPylLX8EeMSXNzcLElxcSyrbx+XCHckRpknauegyzHA7knvX3cKcKbbirN6vzZ5lfE1sSoRrTclBcsbtuyu3ZX2V23bzZe0eIyXTo9jLMPsdzJEqKjHKxOd+HBVlUglscgAlcMAawxVTkimmlrG+/VpW02bvp0va+h6mS4Z1a7lKm5e5NxslvGEnzJSTUoxteS35buPvJHuk+p+EdN0G5uPhtfBrgXFvbR3RtZJLWx5gjlZp3jZY1MSISWJHyAgE5z8D9XxmIrRWYrSzdrpOXxNJJNX95uySW9j+oZZjkWVZbOfCc71OeEFPllKnS1pU5ylUlGShH2cYuTlJ/CpK7vfRm8IeAfi7ppv3vbS51JUVpNQ06LyJRu4TzIyzE5CYAfLBem3NY08djsiq+zjfk/llqvOzsvw073O3F8McNeJWC+tTnGVeyvVpLklrpHni3K90tFO7ttymfo978TPBuqReGb7RbrVYTKoGowW8kkVxGwCrJJIAfLdQoDAg7gBnHLv04qhl2PpPE0ZKD/AJW0mn1SWl0+munT+VeVk2P4t4Zx0Mmx1CVeF0lWjGUoyi9FKcldwlHRSunzW1a1qS9hMWBkj+tfMKmj9pUupD4fg8Pr8V/AL+IrH7TZ6jqV14fnhJO2WK/sbiBkYDqCxjz7A1y5t7ZZLjXQlaUIxqJ9nSqQmn+DPyTxdw1GeEwWKmryVXk/7dqU58y+bjH7j4gKHcVGeK/dkfx7ckA9jQAYz0x+dAEEBUuaAPrjRUnn8W/AX4WeHrTwto9t450PSIb69m8I6ZqE/wBpur+e3M7NcQs7nCpxuAO3sSTX5dk2TUM9xuZVMbUqtwxEoxtWqwSjyU2klCcVu306+h9JjsXUwUMPCiopOnFv3YvXXq0+x1Xxt+Cvxn0P4a+MzonxM8G3XgcPo8erWkfhO00TUL1pBpdzExS0t2XbC+r2gybgMf3mFIyD9Bl/CmUYbF0cwhGbqwTcHOrVny80bS0nOS1Ts9PyMswrY+Ea2GnKLhGXJK0Yq7Tlbon9lv8A4c524/4JufHHQj4llTxx8Ovt3hjUTo4gk1KdHvrw6bDfhLYy2wQnyJw29zGI/KldmRIzIPqKs6c4ONTb+vw7nn4L6zRrxlh7c/TRPXyurc38ttb25fescxZ/svftJ+J/gdr/AMUtVu9GstK0C3nt7nQNTs2i1QQaeblJGSL7MUiMRgvVJaRJB5Dg/wAAPHTwGDoVU4RfNdveX2t766p222PosTxVn2Y4KVGvUjKlyxjZwp6KnpHl926lHnspRtJp7tLTZ0r9gT9p6z8S6d4aXWPCjrqOlnX9Ukm1D7dZaNKZDCY9Qj8qTy5/OR4/MWN0zDcESFbedo96lHDVldrVaXV07bWurO3lsedgs2zXL5OMJ6P33GajKLbSfO41E4uTW0muZ6W3V9XSP2b/ANra/wDHfhf4ZaH4/wDBM15r3hSPxYs9pLDJY6bpr7Fj82WO2KyFnkjjUW3nqxdSpK5Yee8kyx3coP8A8Clv959ZS8UeMoKNOliUlbRKnSSsv+3LJKz9LX0Oi/ZV+HeofGjwv8T734ifF250C/8Ah/c28M2o2b6dPplvA6XRkuWaOCUXSobUkCKVEdTkSYINRVyHLoNJUr3/AL0v8zbD+K3FtXn9pjbWV9KVJ63S/lWmu+vozWT9kf8Aas1Tx54k8F6V8UfCtvBoEVtdR6jqcaRx3NrObtVZhbW84geN9Puo5VdgqyR4V5AyOwslytxUnT/GX+ZUvE7jWNWVKOLTas/4dNXTtZq8E3e6a0vZ3tvbl/h/+zV+1h8QfCujfE7wv4+8I/2xp5u9b0vw9dDy9WM1ldTWjFYWtfKLCWNl+aTYpkj3lSworcP5VWo1MJUpe7UjyyXNLWL6aO623VvU8XNuOeJc/wANCOPxHPCMuaK5acfeit9Ipuylt1vtpo++8CftgWGjR+I01z4fXVm3ga28fMyaBpUO2xnV5BbFprJEF0IYbmYx52+XbTEMxUKfI/1DyC9nGpf/AK/1/wD5YfPf2rj5JyTVkrv3IaK9tfd7tL1PIv2ubm2vNW+Gutx2FpbT6z8OdH1G9NtbRQLLcSvO0khWJVTcT12qB0AAAArk8O4ewpZjhlJuNPFVYx5pOTUUoWV5NvTzZee3k8PUaV5U4t2SWt30Wh4CJx/exX6GeGJCCsvQ0AfZ/wALPBvjTxh8av2ZNU8L+Eda1iz0Gw8N3Wq3FhYS3EVhbrrt1mWdkUiJAsbnc2BhSc8GvieE4qOKzX/sJf8A6
apHt5xthv8Ar3H82fV3xV8Aa38UPAXxZ+Hfw7Ww1LU21bQLYW51G3gMcU1h4TkgcmWRQBILKcJz87RlVy3FfS0/ccG9kv0Rri26v1mmvidRSt1svacz+V1c9ivrDwxq3xF8SP4i1P8As6S1+K9tfaRM8rQkXqeDLPduUwuvyWZvZgJCi+ZBGrbgxQ3Jtx0OTDRp+0XPdtN2SV030V7rS9lpur2a0Z53ZW+keJbTw/8ABnxP8RbTRfiZ8RvA3ieSTwydIuQbf/hI7uG81R/NyY1ezntNTRIXbzCFUMQykvUpSaclbf8A4f8AUUadKE406qafK2/OVm4NeT9z1Tuh+nz63rOveELj4t6Fa20XjbwDdWXxdge7QQaXapDIYLqWUnzbZ3nu9Yt2RdgjiN3ghLDzhCsr26PT5/0/RXKlzStZ6yh7y2Vlt26KLW/M7atysct4u0C38UfCfX/hJ8I/E00+r698DvBz+Gr24i+zSXunWZvEuHkLEJAJoLyC3fcwCC9yxEayMt8/vXa6/wBf5ERpylTUIPVxdl/N723rpdd2rLVpPxX/AIJ5+BJL34XftFfD7xlcy+GBNa2elazPfIkLaSBbaqkzzLO8YXyjncHZcYOSK0quzi1r/wAMzDCQjONROSXu9b/zR7Jn0bpPxa+BWp+Cx48fx7c+EfC8ev8Ah/wvbXt3p91dx6zaadenUbZY1EaSpEzjU7PzpAQyxMzgOSgyjF3s/O/zen+R01pxVP29NNJtKMv8EUn10bvF7uz08ztbr4bweHdX07U9Z0/TNQ8b2i+INR8MWEmoyQxSanrV7qN1JanaFL7LaWZGk3fIizSiN9gKKTk3rs0k/wAbnVGFGjGMqc9YzqOL6PljFxet9XZWVtdFdHjfx81jWfjz+zJY+AbTV5U+I+ufDHw/4uniggRRr1urQSvC+1Pv/aj5cUEa7zPd24Rlje5Wq5uSpzPa9vwX/DHEqanh1Tpr3rOXqouSaVr7L3m7pWTvdqJ+ff7VbyEfCEEt/wAkr0HP5z18bwJbmzT/ALC635QOzPVphf8Ar1D9Tw+JGK53Hr2r748EvOQkuAOnfNSgPp/4f/tRaR4E0bQD4a+IHibw3qFt4Wi8M6xaL4L03WrO+jS8uLhW/wBKukGMzgfcB+U84OK+Fo5bnmWYvGSw1GjUp1qvtE5VZxa9yEbOKozX2L3T6n0Msdhaio1FUnCcIqPupeez5k+pzviL4r+HvFXjXX/iRq/7QnxU/wCEi8U27Wer39p4VtbRry2ZFjaB1g1NVMJREUx42YVRjArvWJ4k2+q0P/B9T/5nOL2eXrX2s/8AwBf/ACZreH/Edl8Utd/scftgfEWx1h71de0658VwSW1nLrUESxW8z3KahMYbgIiItwV3KiBQSQiNwZnnXEOU0frdfAQqQi1zKlVlOaj1koyow5rdk7vyV2t8Ng8Dip+xp12pPbmikr9rqTt9342RQ8bRfFPwd45s/Evxf+Pfj7wz8QtHt1trW9vZ7q5voraQSDNreRzsz253zqSjAAtIrAEkHsy7iZZ1SWIymEa1J31UrWfaSaTjKzWj110utT67BcKcPSoKWZ5jLD1dE4SpOW99YuMnzQ0eqtb7STav59bpqF1cXtzqXxO8YwRa1aNov9oNbySRapp1uqLFbsxnDSR/6PCiQEMB5cQAG0bfTlmOIS9ynFtWuuazV/8At3pu3toOPCGUqXNiMVVjTk5qE/ZJxmo9V+8TvJ6KNnK7V1vbTsPGusBvD/iqy/aE8eSan4WgWw0eNROt7o9oVaPZbsboCKIIpUrC2ArKCACcW8ZjVU5JUIpd+bR26fDfr1S6nPHh7hmeEWJpZhUlK6vTVBc8U21d3q8rWl3yyk0mrpHovw28S6z8PPB/jDwV4O+OVromi+L2m0jXre80G1JvFVJImVZZmZlHlyvzGwI8wZwSK4aub4uLi1h+a6UtG3o++h9HhPD7h3ERquWbqnyzlTfPCMbuL1tepZp6NO+zV7Mc/gDxVN8KpdR8efGbxHoPwn0Mpc6JBqFtOU1HUJA7qNN055lVtyyyN52UBMkh5USuvj4njSf12GVZdQVfFN+9FT92nFaN1JqLUWv5Um/m4p/GZlw7gMLOVXD4lyw62nKHK5y7QhzNtf3ny630srnPal8eL3Vbmxu7/wDa2+PNxcaZO1zZSy2bM9tMYniMkZOr5RjHLIhYYJV2XoxB9lV+Iv8AoFof+D5//M54Hs8t/wCfs/8AwCP/AMsOc1bx14Y1uSKbW/2iPjBfvb6a+jRPdaLHKY9PYYa1UtqxxCRwYx8pHUVX1jiL/oFof+D5/wDzOHs8t/5+z/8AAI//ACwxvj38Q/CnxBv/AAbH4OXVWsvCvhDTvDTzalbR28s8ts0uZRHHLKFVg6nG8kHI5wCcOE8mxmUU8VLHcvPWrTq2g3JJTUdLuMW2rPoi82xlHFuiqF7QhGF2km7N62TffueaRA7fxr6w8kmuZDvJ755pICAyEnnpTAnjnLLjPSoAVXPc1SA9d8H/ALT3irQ/Dkfw/wDiFoWlfEXwam1E0nX1LyWaCMx/6HcjMlswjJVGG4Rj7gU5z8RmnAmDxOKeZZZUlhcS9507Wk9/3kH7s1fV7N9Wz28LntalSWHxMVVp9pbr0e6/G3Sxot/wxV4z+ZX+I3w41C+UEqRDq2kaewHIXpdSqcdTg5PpXHD/AF7y73ZfV8VCPX3qVSXro6cX6aGrWR4l3XPSfylFf+3MSH4e/sneHWbUrn9qDX9fgVCs2m6L4OubK7uUPVFluHMS/wDAhjims54xxX7uOVQpv+adeMor1UI833FU8LlVBuX1uTW1owabT3V27epdsvjR+zz8NAsvwg+DV74j1VQJINZ8e3KTG3fcMoLK3xC4CrlZNyuGY9QADlLhvibPVy53j1Sp7Onhk43XnUneWvWNrNDjmOWYKzwdBzl/NUadv+3Vp6PSx5T8RPib48+KuuHxH498SXWr3xRY0eXakcSAYCxxoAka8ZIUDJJJySTX2WTZFl3D2G+qZbSVOHW27feTd3J+bb002PHxuPxGYVPa4mbk/wAF6LZfI5Qkjv8A/rr1jkGg5+lADsZ5NAD0OFxyPpQA2cHJzjNAERHGcUAKrEDHalYCD7Y0chwuQPemA17suc7MfjQBGZs/w/rQAnm+360APS5Kfw5/GgCQX7Yx5f60ASLN5ig4xkUASIRnAoAlAA5P1oATzFXjg/jQBGXyfmNAClc9KAGy/Ip9+1AGe/3jQAlABQAUAFABQBPGBtFAFqI+4oAmHKggUAVpJcORkCgCKRiKABJHUnDUASynK80AMREIyVB+ooAlMMXTy16egoAUxRbSfLX8qAIWRMn5F/KgCJguPuj8qAGgAjoOnpQAoJB4+lAEqscg0AWlJ+7QBSm5kNAH/9k=","title":"JURASSIC PARK","desc":"Welcome to Jurassic Park!","content":"FROM llama3\nSYSTEM \"\"\"You are a narrator who plays a fictional Jurassic Park themed text roleplay game with the person talking with you who is the player. You start with the player arriving at Jurassic Park with the characters of the original film. The goal of the player and the NPC characters is to survive Jurassic Park. Don't give numbered options to the player. Instead, have the NPC characters suggest options in natural dialogue.\"\"\"","suggestionPrompts":[{"content":"start"}],"categories":["writing"]}]

As you can see, quite a lot has changed. And Open Web UI no longer knows what to do with this.

**Reply by Terristen** [reply]
Glad to see I'm not the only person not happy with the new "improved" workspace tools. In addition to this issue, I'm having the following issues in my situation:

Scenario: 
1 PC with Ollama for windows installed locally, Open WebUI (older version) running as docker image
1 PC with Ollama for windows installed locally. Open WebUI (latest) running as docker image

Old setup behaves as such:
- modelfiles have a RAW option which allows me to specify a file location as the FROM value, pointing to the GGUF files that the modelfile should use
- all modelfiles created in the workspace show up in output when doing an ollama list in terminal
- No need (maybe no tool) to upload a GGUF because I can simply point to the location on disk
- can use curl, Web-Request, and xhttp to call the API manually

New setup behaves as such:
- No more RAW option where I could specify the file location of the GGUF to use
- Have to upload the GGUF using the extremely broken upload tool (experimental) which sometimes just doesn't work(with no obvious error message)
- Creating modelfiles in the Workspace with a base-model specified works in interface but cannot use curl, Web-Request, or xhttp to call the API and request that model
- performing ollama list shows only the basemodel in the list, not any of the modelfiles created in interface.

I need to be able edit and call the created modelfiles as a webservice call to ollama. Why was this changed?

**Reply by justinh-rahb** [reply]
RAW mode still exists chief..

![Screenshot 2024-06-25 at 4 10 32 PM](https://github.com/open-webui/open-webui/assets/52832301/5f636505-0628-479e-a5a1-111337975d03)

**Comment by justinh-rahb** [comment]
We're not fielding any more questions suggestions or complaints about this issue. We've moved on from Modelfile, for the better. There is still a way to use them in **Admin Settings > Models**, and you can create/upload them to your Ollama instance manually and they'll be fully usable in WebUI as before too.
